### PR TITLE
2313 clarify data plane vs control plane

### PR DESCRIPTION
**Pull request by micheleRP**
fixes https://github.com/redpanda-data/documentation-private/issues/2313

This clarifies our doc explaining Redpanda Cloud architecture wrt organization & namespace + control plane & data plane. It also updates the create cluster info with the flattened navigation, how you now create clusters from the Clusters page instead of selecting a namespace first. 

See previews: 

- [Redpanda Cloud architecture](https://deploy-preview-361--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/cloud/cloud-overview/#redpanda-cloud-architecture)
- [Create BYOC on AWS](https://deploy-preview-361--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/cloud/create-byoc-cluster-aws/)
- [Create BYOC on GCP](https://deploy-preview-361--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/cloud/create-byoc-cluster-gcp/)
- [Create Dedicated Clusters](https://deploy-preview-361--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/cloud/create-dedicated-cloud-cluster-aws/)

### CODE REVIEW

File: modules/deploy/pages/deployment-option/cloud/create-dedicated-cloud-cluster-aws.adoc [hunk 1]
```diff
@@ -3,14 +3,11 @@
...
 
 == Create a Dedicated cluster
 
-. On the Home page, select your namespace, or create a new one. 
-+
-Namespaces help you organize clusters and resources. For example, you can create namespaces for different teams or projects. A default namespace is created with every organization. When an organization has more than one namespace, you can select the namespace where you want to create a new cluster. 
-. Click *Create cluster*, then click *Create Dedicated cluster*.
+. On the Clusters page, click *Create cluster*, then click *Create Dedicated cluster*.
 +
 Enter a cluster name, and select the provider (AWS or GCP), region, availability zone type (single AZ or multi AZ) and zones, throughput tier, and Redpanda version.
```
Comment (frankieshakes) [inline modules/deploy/pages/deployment-option/cloud/create-dedicated-cloud-cluster-aws.adoc]: @micheleRP We should also add the bit about selecting a namespace for Dedicated, as we do for BYOC and Serverless.

Comment (micheleRP) [reply modules/deploy/pages/deployment-option/cloud/create-dedicated-cloud-cluster-aws.adoc]: fixed, thanks @frankieshakes!

File: modules/deploy/pages/deployment-option/cloud/cloud-overview.adoc [hunk 2]
```diff
@@ -198,37 +198,40 @@ BYOC with customer-managed VPC::
 --
 =====
 
-== Cloud architecture
+== Redpanda Cloud architecture
```
Comment (greg-szrama) [inline modules/deploy/pages/deployment-option/cloud/cloud-overview.adoc:192]: This header reads awkwardly to me. Is it the product "Redpanda Cloud" architecture? Is it the Redpanda "Cloud architecture"? I may be over-analyzing it but I feel there's a better way to phrase this to get the product name in there.

File: modules/deploy/pages/deployment-option/cloud/cloud-overview.adoc [hunk 3]
```diff
@@ -198,37 +198,40 @@ BYOC with customer-managed VPC::
...
 
-Redpanda Cloud uses a control plane and data plane architecture. The control
-plane configures and maintains cluster agents, which are
-self-contained and independent of the control plane.
+When you first sign up for a Redpanda account, Redpanda creates an organization for you. Your organization contains all your Redpanda resources, including your clusters and networks. Within your organization, Redpanda creates a default namespace as a place to group your resources. You can rename this namespace, and you can create more namespaces. For example, you may want one namespace for a certain team, or you may want different namespaces for production and testing. 
+
+For high availability, Redpanda Cloud uses a control plane and data plane architecture. The control plane enforces rules in the data plane, and the data plane is where your Redpanda cluster resides. The term _data plane_ is used interchangeably with _cluster_.
```
Comment (greg-szrama) [inline modules/deploy/pages/deployment-option/cloud/cloud-overview.adoc]: Would it be good to include a reference to our High Availability document?  Maybe as a 'see also' link at the end of the section?  :)

File: modules/deploy/pages/deployment-option/cloud/cloud-overview.adoc [hunk 4]
```diff
@@ -198,37 +198,40 @@ BYOC with customer-managed VPC::
...
+
+For high availability, Redpanda Cloud uses a control plane and data plane architecture. The control plane enforces rules in the data plane, and the data plane is where your Redpanda cluster resides. The term _data plane_ is used interchangeably with _cluster_. 
+
+The following diagram shows a BYOC architecture, where you deploy the Redpanda data plane in your own VPC. Redpanda manages it on the control plane with an agent. 
+
+image::shared:d_c_plane.png[Data plane and control plane]
+
+In the user interface breadcrumbs, when you're at the organization (org) level or the namespace level, but you haven't yet selected a cluster, you're in the control plane. This is where you can select, create, and delete clusters, namespaces, and networks. When you're at the cluster level working with topics, consumer groups, and connectors, you're in the data plane.
```
Comment (greg-szrama) [inline modules/deploy/pages/deployment-option/cloud/cloud-overview.adoc:211]:
```suggestion
In the user interface breadcrumbs, when you're at the organization (org) level or the namespace level, but you haven't yet selected a cluster, you're in the control plane. 
```

This reads a little awkwardly. Can you rephrase it a little?

Comment (greg-szrama) [inline modules/deploy/pages/deployment-option/cloud/cloud-overview.adoc]: This reads a little awkwardly to me. The reference to breadcrumbs doesn't feel necessary. You might rephrase this a little for clarity.

File: modules/deploy/pages/deployment-option/cloud/cloud-overview.adoc [hunk 5]
```diff
@@ -198,37 +198,40 @@ BYOC with customer-managed VPC::
...
 
 === Control plane
 
-The control plane is a separate entity that enforces rules in the data plane. The
-control plane is where a good deal of your Redpanda cluster management, operations,
-and maintenance takes place. While Redpanda Cloud clusters are configured and
-maintained in the control plane, they are self-contained and remain available
+The control plane is a separate entity where a good deal of cluster management, operations,
+and maintenance takes place. Clusters are configured and
+maintained in the control plane, but they are self-contained and remain available
```
Comment (greg-szrama) [inline modules/deploy/pages/deployment-option/cloud/cloud-overview.adoc]:
```suggestion
The control plane is a discrete entity where you perform cluster management, operations, and maintenance . Clusters are configured and maintained in the control plane, but they are self-contained and remain available
```
I offered a suggestion for rephrasing the first sentence for clarity. I'm not quite sure what the second sentence is trying to convey. Can we add a little more description here? I'm not understanding what it means for clusters to be self-contained and available.

File: modules/deploy/pages/deployment-option/cloud/cloud-overview.adoc [hunk 6]
```diff
@@ -198,37 +198,40 @@ BYOC with customer-managed VPC::
...
 even if the network connection to the control plane is lost.
 
 === Data plane
 
-The data plane is where your Redpanda cluster resides in Redpanda Cloud; it is
-self-contained and managed by an agent that receives cluster specifications from the
-control plane. Redpanda Cloud clusters are interchangeably referred to as
-data planes. All network connections into the data plane take place through
-either a public endpoint or a Redpanda Cloud VPC peering network connection.
+The data plane is self-contained and managed by an agent that receives cluster specifications from the control plane. All network connections into the data plane take place through
```
Comment (greg-szrama) [inline modules/deploy/pages/deployment-option/cloud/cloud-overview.adoc]: Can we explain what it means to be self-contained? (see above on Control plane too).

File: modules/deploy/pages/deployment-option/cloud/cloud-overview.adoc [hunk 7]
```diff
@@ -198,37 +198,40 @@ BYOC with customer-managed VPC::
...
+either a public endpoint or a VPC peering network connection. Sensitive data and credentials never leave the data plane. 
 
 === Agent
 
-The Redpanda Cloud agent, also known as the data plane agent, provisions,
+The control plane configures and maintains cluster agents, which are
+self-contained and independent of the control plane. The agent provisions,
```
Comment (greg-szrama) [inline modules/deploy/pages/deployment-option/cloud/cloud-overview.adoc]: Is it one agent or multiple agents operating here?

File: modules/deploy/pages/deployment-option/cloud/cloud-overview.adoc [hunk 8]
```diff
@@ -198,37 +198,40 @@ BYOC with customer-managed VPC::
...
 configures, and maintains all the resources in the cluster, from
 the network all the way up to the cluster and connectors infrastructure.
 Data plane resources include cloud infrastructure, Kubernetes, and data plane software
```
Comment (greg-szrama) [inline modules/deploy/pages/deployment-option/cloud/cloud-overview.adoc]: Should this description of Data plane resources be here?

File: modules/deploy/pages/deployment-option/cloud/cloud-overview.adoc [hunk 9]
```diff
@@ -198,37 +198,40 @@ BYOC with customer-managed VPC::
...
-Redpanda Cloud uses a control plane and data plane architecture. The control
-plane configures and maintains cluster agents, which are
-self-contained and independent of the control plane.
+When you first sign up for a Redpanda account, Redpanda creates an organization for you. Your organization contains all your Redpanda resources, including your clusters and networks. Within your organization, Redpanda creates a default namespace as a place to group your resources. You can rename this namespace, and you can create more namespaces. For example, you may want one namespace for a certain team, or you may want different namespaces for production and testing. 
+
+For high availability, Redpanda Cloud uses a control plane and data plane architecture. The control plane enforces rules in the data plane, and the data plane is where your Redpanda cluster resides. The term _data plane_ is used interchangeably with _cluster_. 
+
+The following diagram shows a BYOC architecture, where you deploy the Redpanda data plane in your own VPC. Redpanda manages it on the control plane with an agent. 
+
+image::shared:d_c_plane.png[Data plane and control plane]
```
Comment (greg-szrama) [inline modules/deploy/pages/deployment-option/cloud/cloud-overview.adoc]: Can we add the Agent to the diagram, showing how it orchestrates between the control and data planes?

File: modules/deploy/pages/deployment-option/cloud/cloud-overview.adoc [hunk 10]
```diff
@@ -198,37 +198,40 @@ BYOC with customer-managed VPC::
...
-The Redpanda Cloud agent, also known as the data plane agent, provisions,
+The control plane configures and maintains cluster agents, which are
+self-contained and independent of the control plane. The agent provisions,
 configures, and maintains all the resources in the cluster, from
 the network all the way up to the cluster and connectors infrastructure.
 Data plane resources include cloud infrastructure, Kubernetes, and data plane software
 artifacts.
 
-Agents can be bootstrapped in two different ways: automatically by a machine
+Agents can be bootstrapped automatically by a machine
```
Comment (greg-szrama) [inline modules/deploy/pages/deployment-option/cloud/cloud-overview.adoc]: There's some passive voice here, and I'm not sure what "bootstrapped automatically by a machine" means. Can we expound on that a bit?

File: modules/deploy/pages/deployment-option/cloud/cloud-overview.adoc [hunk 11]
```diff
@@ -237,10 +240,10 @@ specifications that provision, configure, and maintain Redpanda Cloud clusters.
 The agent is in constant communication with the control plane,
 receiving and applying cluster specifications and exchanging cluster metadata.
 Agents are authenticated and authorized through opaque and ephemeral tokens, and
```
Comment (greg-szrama) [inline modules/deploy/pages/deployment-option/cloud/cloud-overview.adoc]:
```suggestion
Redpanda uses opaque and ephemeral tokens to authenticate agents.
```
What's the technology used for these tokens? OAUTH? JWT?

File: modules/deploy/pages/deployment-option/cloud/create-byoc-cluster-aws.adoc [hunk 12]
```diff
@@ -3,16 +3,12 @@
 :page-aliases: cloud:create-byoc-cluster-aws.adoc
 :page-cloud: true
 
-To create a Redpanda cluster in your virtual private cloud (VPC), log in to https://cloud.redpanda.com[Redpanda Cloud^]. Enter your organization name, and click *Continue*. 
+After you log in to https://cloud.redpanda.com[Redpanda Cloud^], you land on the *Clusters* page, which lists all the clusters in your organization. When you create new clusters, if your organization has more than one namespace, you select the namespace where you want to create the new cluster.
```
Comment (greg-szrama) [inline modules/deploy/pages/deployment-option/cloud/create-byoc-cluster-aws.adoc]:
```suggestion
Upon logging in to https://cloud.redpanda.com[Redpanda Cloud^], you are shown the *Clusters* page. This page lists all the clusters in your organization. If your organization has more than one namespace, you can select the target namespace when creating a new cluster. 
```

File: modules/deploy/pages/deployment-option/cloud/create-byoc-cluster-aws.adoc [hunk 13]
```diff
@@ -3,16 +3,12 @@
...
 == Create a BYOC cluster
 
-. On the Home page, select your namespace, or create a new one.
-+
-Namespaces help you organize clusters and resources. For example, you can create namespaces for different teams or projects. A default namespace is created with every organization. When an organization has more than one namespace, you can select the namespace where you want to create a new cluster. 
-. Click *Create cluster*, then click
-+
-Enter a cluster name, and select the provider (AWS), region, availability zone type (single AZ or multi AZ) and zones, throughput tier, and Redpanda version.
+. On the Clusters page, click *Create cluster*, then click *Create BYOC cluster*.
+. Enter a cluster name, and select the namespace, provider (AWS), region, availability zone type (single AZ or multi AZ) and zones, throughput tier, and Redpanda version.
```
Comment (greg-szrama) [inline modules/deploy/pages/deployment-option/cloud/create-byoc-cluster-aws.adoc]:
```suggestion
. Enter a cluster name then select the namespace, provider (AWS), region, availability zone type (single AZ or multi AZ) and zones, throughput tier, and Redpanda version.
```

File: modules/deploy/pages/deployment-option/cloud/create-byoc-cluster-gcp.adoc [hunk 14]
```diff
@@ -3,18 +3,15 @@
 :page-aliases: cloud:create-byoc-cluster-gcp.adoc
 :page-cloud: true
 
-To create a Redpanda cluster in your virtual private cloud (VPC), log in to https://cloud.redpanda.com[Redpanda Cloud^]. Enter your organization name, and click *Continue*. 
+After you log in to https://cloud.redpanda.com[Redpanda Cloud^], you land on the *Clusters* page, which lists all the clusters in your organization. When you create new clusters, if your organization has more than one namespace, you select the namespace where you want to create the new cluster.
```
Comment (greg-szrama) [inline modules/deploy/pages/deployment-option/cloud/create-byoc-cluster-gcp.adoc]:
```suggestion
Upon logging in to https://cloud.redpanda.com[Redpanda Cloud^], you are shown the *Clusters* page. This page lists all the clusters in your organization. If your organization has more than one namespace, you can select the target namespace when creating a new cluster. 
```

File: modules/deploy/pages/deployment-option/cloud/create-byoc-cluster-gcp.adoc [hunk 15]
```diff
@@ -3,18 +3,15 @@
...
 == Create a BYOC cluster
 
-. On the Home page, select your namespace, or create a new one.
+. On the Clusters page, click *Create cluster*, then click *Create BYOC cluster*.
 +
-Namespaces help you organize clusters and resources. For example, you can create namespaces for different teams or projects. A default namespace is created with every organization. When an organization has more than one namespace, you can select the namespace where you want to create a new cluster. 
-. Click *Create cluster*, then click *Create BYOC cluster*.
-+
-Enter a cluster name, and select the provider (GCP), region, availability zone type (single AZ or multi AZ) and zones, throughput tier, and Redpanda version.
+Enter a cluster name, and select the namespace, provider (GCP), region, availability zone type (single AZ or multi AZ) and zones, throughput tier, and Redpanda version.
```
Comment (greg-szrama) [inline modules/deploy/pages/deployment-option/cloud/create-byoc-cluster-gcp.adoc]:
```suggestion
Enter a cluster name then select the namespace, provider (GCP), region, availability zone type (single AZ or multi AZ) and zones, throughput tier, and Redpanda version.
```

File: modules/deploy/pages/deployment-option/cloud/create-dedicated-cloud-cluster-aws.adoc [hunk 16]
```diff
@@ -3,16 +3,13 @@
 :page-aliases: cloud:create-dedicated-cloud-cluster-aws.adoc, deploy:deployment-option/cloud/provision-a-dedicated-cluster/index.adoc
 :page-cloud: true
 
-To create a Dedicated Cloud cluster, log in to https://cloud.redpanda.com[Redpanda Cloud^]. Enter your organization name, and click *Continue*. 
+After you log in to https://cloud.redpanda.com[Redpanda Cloud^], you land on the *Clusters* page, which lists all the clusters in your organization. When you create new clusters, if your organization has more than one namespace, you select the namespace where you want to create the new cluster.
```
Comment (greg-szrama) [inline modules/deploy/pages/deployment-option/cloud/create-dedicated-cloud-cluster-aws.adoc]:
```suggestion
Upon logging in to https://cloud.redpanda.com[Redpanda Cloud^], you are shown the *Clusters* page. This page lists all the clusters in your organization. If your organization has more than one namespace, you can select the target namespace when creating a new cluster. 
```

File: modules/deploy/pages/deployment-option/cloud/create-dedicated-cloud-cluster-aws.adoc [hunk 17]
```diff
@@ -3,16 +3,13 @@
...
 == Create a Dedicated cluster
 
-. On the Home page, select your namespace, or create a new one. 
+. On the Clusters page, click *Create cluster*, then click *Create Dedicated cluster*.
 +
-Namespaces help you organize clusters and resources. For example, you can create namespaces for different teams or projects. A default namespace is created with every organization. When an organization has more than one namespace, you can select the namespace where you want to create a new cluster. 
-. Click *Create cluster*, then click *Create Dedicated cluster*.
-+
-Enter a cluster name, and select the provider (AWS or GCP), region, availability zone type (single AZ or multi AZ) and zones, throughput tier, and Redpanda version.
+Enter a cluster name, and select the namespace, provider (AWS or GCP), region, availability zone type (single AZ or multi AZ) and zones, throughput tier, and Redpanda version.
```
Comment (greg-szrama) [inline modules/deploy/pages/deployment-option/cloud/create-dedicated-cloud-cluster-aws.adoc]:
```suggestion
Enter a cluster name then select the namespace, provider (AWS or GCP), region, availability zone type (single AZ or multi AZ) and zones, throughput tier, and Redpanda version.
```

**Review by greg-szrama — CHANGES_REQUESTED** [review]
There are a few places I think can use some clarification. It's a good rundown on the feature, though!

File: modules/deploy/pages/deployment-option/cloud/cloud-overview.adoc (hunk 2 above)
Comment (micheleRP) [reply modules/deploy/pages/deployment-option/cloud/cloud-overview.adoc:192]: @greg-szrama: Yes, the product name is Redpanda Cloud.

File: modules/deploy/pages/deployment-option/cloud/cloud-overview.adoc (hunk 8 above)
Comment (micheleRP) [reply modules/deploy/pages/deployment-option/cloud/cloud-overview.adoc]: fixed!

File: modules/deploy/pages/deployment-option/cloud/cloud-overview.adoc (hunk 10 above)
Comment (micheleRP) [reply modules/deploy/pages/deployment-option/cloud/cloud-overview.adoc]: rephrased

File: modules/deploy/pages/deployment-option/cloud/cloud-overview.adoc (hunk 6 above)
Comment (micheleRP) [reply modules/deploy/pages/deployment-option/cloud/cloud-overview.adoc]: rephrased

File: modules/deploy/pages/deployment-option/cloud/cloud-overview.adoc (hunk 3 above)
Comment (micheleRP) [reply modules/deploy/pages/deployment-option/cloud/cloud-overview.adoc]: We already have a lot of links on this page. I don't want to distract users away from this high-level info into the details of single-AZ/multi-AZ setups.

**Review by greg-szrama — APPROVED** [review]
Looking good! Looks like I have a few outstanding questions there but nothing new comes up to my eye.

File: modules/deploy/pages/deployment-option/cloud/cloud-overview.adoc (hunk 7 above)
Comment (micheleRP) [reply modules/deploy/pages/deployment-option/cloud/cloud-overview.adoc]: one agent

**Review by c4milo — APPROVED** [review]
